library/testing/TestSDKCamera.cxx

```diff
@@ -179,8 +179,30 @@ int TestSDKCamera(int argc, char* argv[])
     return EXIT_FAILURE;
   }
 
+  // Test getAzimuth
+  f3d::angle_deg_t addAzimuth = cam.getAzimuth();
```

Comment:
strange name, why not call the var "azimuth" ?

Reply:
That's the name we decided with @Meakk and @snoyer. If not appropriate I can change it.

Reply:
I mean the name of the variable. not the name of the method.

Reply:
Oh that's there just to clarify it. I can remove it.